### PR TITLE
Add ProjectFreeTV Provider (Squash Merge)

### PR DESCRIPTION
**Pull request by Teekzilla**
Added TV Provider: ProjectFreeTV

### CODE REVIEW

File: src/scrapers/providers/index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ module.exports = exports = {
         new (require('./tv/SeriesFree'))(),
         new (require('./tv/GoWatchSeries'))(),
         new (require('./tv/SwatchSeries'))(),
+        new (require('./tv/ProjectFreeTV'))
```
Comment (ddivad) [inline src/scrapers/providers/index.js]: Should this be `new (require('./tv/ProjectFreeTV'))()` instead of `new (require('./tv/ProjectFreeTV'))` ?

Comment (Teekzilla) [reply src/scrapers/providers/index.js]: Yup, my mistake - good catch!

**Comment by kevincosmos** [comment]
Initial testing on this seems good :)

File: src/scrapers/providers/tv/ProjectFreeTV.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+const Promise = require('bluebird');
+const RequestPromise = require('request-promise');
+const cheerio = require('cheerio');
+const randomUseragent = require('random-useragent');
+const URL = require('url');
+const { absoluteUrl, removeYearFromTitle } = require('../../../utils');
```
Comment (FlyPigsFly) [inline src/scrapers/providers/tv/ProjectFreeTV.js]:
```suggestion
```
Hi noticed this wasn't required

File: src/scrapers/providers/tv/ProjectFreeTV.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
+const Promise = require('bluebird');
+const RequestPromise = require('request-promise');
```
Comment (FlyPigsFly) [inline src/scrapers/providers/tv/ProjectFreeTV.js]:
```suggestion
```

File: src/scrapers/providers/tv/ProjectFreeTV.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+const cheerio = require('cheerio');
+const randomUseragent = require('random-useragent');
+const URL = require('url');
```
Comment (FlyPigsFly) [inline src/scrapers/providers/tv/ProjectFreeTV.js]:
```suggestion
```

File: src/scrapers/providers/tv/ProjectFreeTV.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
+const Promise = require('bluebird');
+const RequestPromise = require('request-promise');
+const cheerio = require('cheerio');
+const randomUseragent = require('random-useragent');
```
Comment (FlyPigsFly) [inline src/scrapers/providers/tv/ProjectFreeTV.js]:
```suggestion
```

**Comment by Teekzilla** [comment]
Just going to leave this here but please squash merge this one!

**Pull request merged**